ethereum/gateway-eth-ts/src/contracts/FlagsStorage.ts

```diff
@@ -18,51 +21,54 @@ export class FlagsStorage {
     flag: BytesLike,
     index: number | BigNumber,
     txParams?: TxBase
-  ): Promise<any> => {
-    return this.contract.addFlag(flag, index, txParams);
+  ): Promise<unknown> => {
+    return this.contract.addFlag(flag, index, txParams) as unknown;
```

Comment:
addFlag returns Promise<string>. why unknown here?

Reply:
ok this.contract is ethers contract?

Reply:
So I assume contract does not specifically refer to `https://github.com/identity-com/on-chain-identity-gateway/blob/2928ce3e6100e47ce0aa39556674be82e114ece8/ethereum/gateway-eth-ts/src/GatewayTs.ts#L158`?